bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1980,10 +1980,19 @@ void ReplicaImp::onNewView(const std::vector<PrePrepareMsg *> &prePreparesForNew
     primaryLastUsedSeqNum = lastStableSeqNum;
     strictLowerBoundOfSeqNums = lastStableSeqNum;
     maxSeqNumTransferredFromPrevViews = lastStableSeqNum;
+    LOG_INFO(GL,
+             "No PrePrepare-s for the new view: " << KVLOG(
+                 primaryLastUsedSeqNum, strictLowerBoundOfSeqNums, maxSeqNumTransferredFromPrevViews));
   } else {
     primaryLastUsedSeqNum = lastPPSeq;
     strictLowerBoundOfSeqNums = firstPPSeq - 1;
     maxSeqNumTransferredFromPrevViews = lastPPSeq;
+    LOG_INFO(GL,
+             "There are PrePrepare-s for the new view: " << KVLOG(firstPPSeq,
+                                                                  lastPPSeq,
+                                                                  primaryLastUsedSeqNum,
+                                                                  strictLowerBoundOfSeqNums,
+                                                                  maxSeqNumTransferredFromPrevViews));
```

Comment:
Same thing here, I think the sync of the active window after View Change is good to be consistent with the convention of View Change logs.